### PR TITLE
Maneo's branch

### PR DESCRIPTION
**Pull request by NthabisengLekolwana**
Please add our trademark tote footer and place the footer at the base

### CODE REVIEW

**Review by NthabisengLekolwana — COMMENTED** [review]
@ManeoMalatsi , please change the colour of the font on the view details' page.

**Comment by ManeoMalatsi** [comment]
> Please add our trademark tote footer and place the footer at the base

@NthabisengLekolwana, It has been fixed please check it

File: README.md [hunk 1]
```diff
@@ -1,3 +1,116 @@
-# MovieAppTH
-Communication Channel
-# testing
```
Comment (XolaniMba) [inline README.md]: @NthabisengLekolwana @ManeoMalatsi @neoThobela @kgaugelomokgalaka  please update the readme file and the the pull request description

**Comment by ManeoMalatsi** [comment]
> @ManeoMalatsi , please change the colour of the font on the view details' page.

@NthabisengLekolwana, it is changed. Thanks.

**Comment by XolaniMba** [comment]
> > Please add our trademark tote footer and place the footer at the base
> 
> @NthabisengLekolwana, It has been fixed please check it

@ManeoMalatsi okay I see the changes has been made, Thanks

**Comment by neoThobela** [comment]
@ManeoMalatsi i just pulled your code everything is working okay

**Comment by neoThobela** [comment]
@Team i just pushed the nav bar please pull and and test

**Review by kgaugelomokgalaka — COMMENTED** [review]
the changes made are good

**Review by XolaniMba — APPROVED** [review]
the app looks good; you can merge

**Review by kgaugelomokgalaka — COMMENTED** [review]
the code is good

**Pull request merged**